recipes/doxygen/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - toolchain
     - flex
     - bison
-    - python  # [win]
+    - python
```

Comment:
Since python is added, there'll be 3 builds. To avoid that, you can add `skip: True  # [win or py !=35]`